src/utils/get-changed-filepaths.ts

```diff
@@ -17,8 +17,17 @@ import { octokit } from '../octokit';
 
 export const getChangedFilepaths = async (pull_number: number, ignore_deleted?: boolean) => {
   const changedFiles = await paginateAllChangedFilepaths(pull_number);
-  const filesToMap = ignore_deleted ? changedFiles.filter(file => file.status !== 'removed') : changedFiles;
-  return filesToMap.map(file => file.filename);
+  const files = Array.from(
+    changedFiles.reduce((acc, file) => {
+      if (ignore_deleted && file.status === 'removed') return acc;
+      acc.add(file.filename);
+      if (file.status === 'renamed' && file.previous_filename) {
+        acc.add(file.previous_filename);
+      }
+      return acc;
+    }, new Set<string>())
+  );
+  return files;
```

Comment:
This is pretty hard to read -- could we just create two arrays and concat them?

Reply:
sure, i was using a hashset to dedupe but in hindsight i don't think there's a possible scenario where there could be duplicates